cfn-resources/project-ip-access-list/cmd/resource/resource.go

```diff
@@ -188,15 +175,14 @@ func Delete(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		currentModel.Profile = aws.String(profile.DefaultProfile)
 	}
 
-	// Create atlas client
 	client, peErr := util.NewMongoDBClient(req, currentModel.Profile)
 	if peErr != nil {
 		return *peErr, nil
 	}
 
 	event := deleteEntries(currentModel, client)
 	if event.OperationStatus == handler.Failed {
-		_, _ = logger.Warnf("Delete deleteEntries error:%+v", event.Message)
+		_, _ = logger.Warnf("Delete error: %+v", event.Message)
 		return event, nil
 	}
 
```

Comment:
@Adelmar92 I see that the resource implements the `LIST` operation. Could you help me understand when this resource is called in the template? I m wondering if we should remove it

Reply:
i understand that the list operation is used with the https://docs.aws.amazon.com/cli/latest/reference/cloudcontrol/list-resources.html